prisma/migrations/20231031014446_initial/migration.sql

```diff
@@ -10,7 +10,7 @@ CREATE TABLE "jira_connect_installation" (
     "base_url" TEXT NOT NULL,
     "display_url" TEXT NOT NULL,
 
-    CONSTRAINT "connect_installation_pkey" PRIMARY KEY ("id")
+    CONSTRAINT "jira_connect_installation_pkey" PRIMARY KEY ("id")
```

Comment:
**note:** @elainefigma, we forgot to rename primary and foreign keys, so, currently, there is a mismatch between the schema definition and the migration script. 

Likely, this happened because the sql file was modified manually instead of being generated from `schema.prisma` by Prisma (`npm run db:migrate:dev`).

This PR will include the fix. Please, pay attention that you might need to reset the database on your side.